packages/forms-web-app/src/controllers/common/enter-code.js

```diff
@@ -208,6 +223,18 @@ const postEnterCode = (views) => {
 		// if handling an email confirmation
 		// session will be in browser so can redirect here and consider email confirmed
 		if (tokenValid.action === enterCodeConfig.actions.confirmEmail) {
+			if (isReturningUser) {
+				const user = await getUserByEmailV2(req.session.appeal.email);
+				if (!user.isEnrolled) {
```

Comment:
is this required? I added an auto enrol to the token endpoint in api?

Reply:
Oh okay, that makes sense, that didn't work locally because we accept the 12345 token without calling the check token endpoint